frontend/sozisel-app/src/components/SessionsList/SessionsList.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../styles/styles";
+
+.SessionsListContainer {
+    // 80px = navbar height
+    // i don't know why, but $navbar-height spacing doesn't work
+    // so i had to hardcoded this value
+    height: calc(100% - 80px);
```

Comment:
because you have to use different syntax to use variables in here, just google how to use variables inside calc in scss